manager/volume.go

```diff
@@ -290,7 +290,19 @@ func (m *VolumeManager) DeleteReplica(replicaName string) error {
 }
 
 func (m *VolumeManager) GetManagerNodeIPMap() (map[string]string, error) {
-	return m.ds.GetManagerNodeIPMap()
+	podList, err := m.ds.ListManagerPods()
+	if err != nil {
+		return nil, err
+	}
+
+	nodeIPMap := make(map[string]string)
```

Comment:
Same here, use `map[string]string{}` instead except you need the length parameter of `make`.